src/pseudopeople/noise_functions.py

```diff
@@ -73,4 +75,34 @@ def generate_phonetic_errors(
     return column
 
 
+def missing_data(
+    column: pd.Series,
+    configuration: ConfigTree,
+    randomness_stream: RandomnessStream,
+    additional_key: Any,
```

Comment:
What is this additional key for? If we're adding an additional key here we will need to add it where we actually call the noise functions in `noise.noise_forms`.

Reply:
We are doing this same noise function on multiple columns.  So we need to make a unique key for each column we process.

Reply:
Is that not what my implementation is doing in `noise.py`?